proto/buf/alpha/registry/v1alpha1/plugin.proto

```diff
@@ -324,6 +340,27 @@ message ListPluginContributorsResponse {
   string next_page_token = 2;
 }
 
+message DeprecatePluginByNameRequest {
+  // The owner of the plugin.
+  string owner = 1;
+  // The name of the plugin.
+  string name = 2;
+  // A message shown along with the deprecation warning for this plugin.
+  // It must be a utf8 encoded string containing 256 or fewer characters.
```

Comment:
We don't include the validation constraints in any other comments in this file, so we should probably remove them from here?